pages/Variations/[imageID].js

```diff
@@ -0,0 +1,70 @@
+import Image from "next/image.js";
+import styled from "styled-components";
+import { StyledA } from "../../components/StyledLink";
+import useSWR from "swr";
+import { useRouter } from "next/router.js";
+
+export default function ChooseFourVariations() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { imageID } = router.query;
```

Comment:
do you work with `useRouter()` in this component?

if not, you should remove those lines of code